src/level/TMXRenderer.js

```diff
@@ -525,8 +525,13 @@
          * the way Tiled places them
          * @ignore
          */
-        adjustPosition: function (/*obj*/) {
-            // TBD
+        adjustPosition: function (obj) {
+            // only adjust position if obj.gid is defined
+            if (typeof(obj.gid) === "number") {
```

Comment:
there is some weird things about tiled object that have a wrong position in orthogonal maps, I suppose it is the same with hexagonal maps ?

@matthewmmorrow did u try yet to add some object to your map in Tiled and see if they were placed properly in melonJS ?